client/src/amplify-ui/sidebar-layout/sidebar-open-button/sidebar-open-button.style.ts

```diff
@@ -0,0 +1,31 @@
+import {css} from "emotion";
+import {MQTablet} from "../../styles/media";
+
+export const sidebarOpenButtonStyle = css`
+  width: 2.5rem;
+  margin-top: 1.75rem;
+  padding: 0.3125rem 0.4375rem 0.125rem 0.5rem;
+  position: absolute;
+
+  border-top: 0.0625rem solid var(--border-color);
+  border-right: 0.0625rem solid var(--border-color);
+  border-bottom: 0.0625rem solid var(--border-color);
+  border-radius: 0 0.3125rem 0.3125rem 0;
+  box-shadow: rgba(0, 0, 0, 0.09) 0.08rem 0.15625rem 0 0.08rem;
+
+  background-color: var(--bg-color);
+  :hover {
```

Comment:
Do you intend to select all child hover states?

Or do you intend to do this?:

```ts
&:hover {
  ...
}
```